weave-js/src/components/Panel2/PanelTable/util.ts

```diff
@@ -42,9 +42,14 @@ export const getColumnCellFormats = (colType: Type): WeaveFormatContextType => {
         }
       : {};
   const stringFormat = {spacing: t === 'string'};
+  const columnFormat = {
+    textAlign: t === 'number' ? ('right' as const) : ('center' as const),
```

Comment:
is `as const` necessary? This a type exported from React for CSS styles if you want to use that. non-blocker though

Reply:
I tried using it, but I got type errors from it, so I used `as const` because it was used, (I assume to resolve the issue), in `numberFormat`

here's the type error:
![image](https://github.com/user-attachments/assets/50a5102a-e4e1-4278-9b78-9ba04ba52655)